src/scss/components/_navbar.scss

```diff
@@ -0,0 +1,62 @@
+//------------------------------------------------------------------------------
+// @component: Navbar
+//------------------------------------------------------------------------------
+// @description
+//
+//------------------------------------------------------------------------------
+
+//
+// @settings
+
+// Layout Variables
+
+
+//
+// @scss
+.c-navbar {
+	overflow: hidden;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+
+	&-logo {
+		@include type(title);
+		line-height: 1;
+		float: left;
+		flex: 1;
+		margin-bottom: 0;
+		padding: 16px;
+	}
+
+	&-menu {
+		list-style-type: none;
+		margin: 0;
+		padding: 0;
+	}
+
+	&-menu-item {
+		display: flex;
+		float: right;
+	}
+
+	&-menu-link {
+		display: inline-block;
+		font-size: 16px;
+		line-height: 3.5;
+		color: #868686;
```

Comment:
this should be a local variable in the future. infact all these should be variables that is why i said to just 100% use the component I already built verbatim. Until you get the hang of building your own